includes/payment-tokens/class-wc-stripe-klarna-payment-token.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * WooCommerce Stripe Klarna Payment Token
+ *
+ * Representation of a payment token for Klarna.
+ *
+ * @package WooCommerce_Stripe
+ * @since 10.0.0
+ */
+
+// phpcs:disable WordPress.Files.FileName
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+class WC_Payment_Token_Klarna extends WC_Payment_Token implements WC_Stripe_Payment_Method_Comparison_Interface {
```

Comment:
I missed this before merging: I think we need to use a Stripe-specific class name here, as it feels like a collision could occur way too easily. Is there a reason you picked a generic name, @wjrosa?
```suggestion
class WC_Stripe_Klarna_Payment_Token extends WC_Payment_Token implements WC_Stripe_Payment_Method_Comparison_Interface
```